### PR TITLE
Subaru: log stock AEB

### PR DESCRIPTION
**Pull request by martinl**
This PR adds stock AEB dash signals logging (from #25345) and adds FCW_Cont_Beep alert to stockFcw

### CODE REVIEW

File: selfdrive/car/subaru/carstate.py [hunk 1]
```diff
@@ -77,6 +77,8 @@ def update(self, cp, cp_cam, cp_body):
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
       ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
+      ret.stockAeb = (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 5) or \
+                     (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert_Msg"] == 6) or ret.stockFcw
```
Comment (sshane) [inline selfdrive/car/subaru/carstate.py:81]: why set when there's a fcw? we usually want to differentiate between AEB and FCW

Comment (martinl) [reply selfdrive/car/subaru/carstate.py]: I reviewed the other brands openpilot long implementations and removed stockFcw from stockAeb

File: selfdrive/car/subaru/carstate.py [hunk 2]
```diff
@@ -77,6 +77,8 @@ def update(self, cp, cp_cam, cp_body):
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
       ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
```
Comment (sshane) [inline selfdrive/car/subaru/carstate.py:79]: FCW_Cont_Beep is 1, should we add that too?

Comment (martinl) [reply selfdrive/car/subaru/carstate.py]: yes, I added FCW_Cont_Beep to stockFcw

**Comment by adeebshihadeh** [comment]
@sshane this good to merge?

**Comment by sshane** [comment]
Not quite, LKAS_Alert_Msg activates too late. Think I found the real AEB actuation bit in the ES_Brake message though

image

**Comment by sshane** [comment]
LGTM! Posted notes in the opendbc PR

File: selfdrive/car/subaru/carstate.py [hunk 3]
```diff
@@ -69,17 +69,21 @@ def update(self, cp, cp_cam, cp_body):
...
       self.cruise_button = cp_cam.vl["ES_Distance"]["Cruise_Button"]
       self.ready = not cp_cam.vl["ES_DashStatus"]["Not_Ready_Startup"]
     else:
       ret.steerFaultTemporary = cp.vl["Steering_Torque"]["Steer_Warning"] == 1
       ret.cruiseState.nonAdaptive = cp_cam.vl["ES_DashStatus"]["Conventional_Cruise"] == 1
       ret.cruiseState.standstill = cp_cam.vl["ES_DashStatus"]["Cruise_State"] == 3
-      ret.stockFcw = cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2
+      ret.stockFcw = (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 1) or \
+                     (cp_cam.vl["ES_LKAS_State"]["LKAS_Alert"] == 2)
+      # 8 is known AEB, but there are a few other values related to AEB we ignore
```
Comment (sshane) [inline selfdrive/car/subaru/carstate.py:82]:
```suggestion
      # 8 is known AEB, there are a few other values related to AEB we ignore
```

**Pull request merged**